sessions/react-global-state/demo-end/pages/index.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+const Flex = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: var(--accent-color);
+`;
+
+export default function HomePage({ countSum, dragonCount, countAverage }) {
+  return (
+    <Flex>
```

Comment:
Both pages have a very different shape, which confuses me a bit. Maybe we can find a way to re-use some styled components and have the same structure on both pages?

We could create an individual component for the counter stats.

Maybe something like this? Or create a styled component with grid/flexbox and use it instead of fragments

```jsx
// index
<>
  <MainWrapper>
    <CounterStats />
  </MainWrapper>

  <NavWrapper>
    <StyledLink>To the counters ➡︎</StyledLink>
  </NavWrapper>
</>
```

```jsx
// overview
<>
  <NavWrapper>
    <StyledLink>⬅︎ Back</StyledLink>
  </NavWrapper>

  <MainWrapper>
    <Counters />
  </MainWrapper>

  <NavWrapper>
    <Navigation />
  </NavWrapper>
</>
```